src/handlers/interactions/chatInputCommands.ts

```diff
@@ -34,6 +35,8 @@ export default async function chatInputCommandHandler(interaction: ChatInputComm
     const countingChannel = document.channels.get(interaction.channelId);
     if (command.disableInCountingChannel && countingChannel) return void interaction.reply({ content: "❌ This command is disabled in counting channels.", ephemeral: true });
 
+    if (command.requireSelectedCountingChannel && document.channels.size === 0) return void interaction.reply({ content: `💥 No counting channel is set up in this server! Create a new one by using \`/channels new\` or link an existing one with \`/channels link\`. New to Countr? Check out the [documentation](<${docsUrl}>) to get started!`, ephemeral: true });
```

Comment:
this is the 3rd instance of this interaction text setup, refactor into a single file of reused messages?

Reply:
however, that is not blocking atm as people are experiencing this error in the wild

Reply:
logic is different in all three different places, and also not a priority atm